src/messaging/ReliableMessageMgr.cpp

```diff
@@ -490,6 +491,29 @@ void ReliableMessageMgr::FailRetransTableEntries(ReliableMessageContext * rc, CH
     }
 }
 
+/**
+ *  Retransmit all pending messages that were encrypted with application
+ *  group key and were addressed to the specified node.
+ *
+ *  @param[in] peerNodeId    Node ID of the destination node.
+ *
+ */
+void ReliableMessageMgr::RetransPendingAppGroupMsgs(uint64_t peerNodeId)
```

Comment:
Please use full names like: `RetransmitPendingApplicationGroupMessages`

Reply:
This function might be left to the later part of MCSP since it is not called in this PR